.vscode/settings.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
what's the reason here?

Reply:
These were messing with my setup (namely: prettier was not running)
I'm not a fan of sharing IDE settings, but if you require it then I'll try to fix this

Reply:
No it's ok